docs/tools/feature-flags/how-to-use-feature-flags.md

```diff
@@ -0,0 +1,113 @@
+
+The feature flag data is stored in a separate, private repository `transcom/milmove-feature-flags`.
+
+As for 2023-09-06, feature flags have not been used for a full end to end feature deployment. This document may have some gaps because of that.
+
+### Feature Flag Evaluation
+
+First, familiarize yourself with the key [Flipt Concepts](https://www.flipt.io/docs/concepts).
+
+Each deployed environment (demo, exp, loadtest, stg, prd) is in its
+own [Flipt Namespace](https://www.flipt.io/docs/concepts#namespaces) and only that single namespace is available in each environment.
+
+Flipt supports two kinds of feature flags:
+ * [Boolean Flags](https://www.flipt.io/docs/concepts#boolean-flags)
+ * [Variant Flags](https://www.flipt.io/docs/concepts#variant-flags)
+
+Boolean flags are for when a feature can only be turned on or off. Variant flags are for when a feature might have multiple options.
+
+The most common [Entity](https://www.flipt.io/docs/concepts#entities) to work with is a user and that is probably how most feature flags will be used: to determine if a particular user should have access to a feature or not.
+
+The [Context](https://www.flipt.io/docs/concepts#context) contains multiple bits of information that can by used by flipt to determine if a feature should be available to a user or not.
+
+### MilMove Backend Feature Flag Service
+
+A [backend service object](../../backend/guides/service-objects/overview.md) has been created for feature flags that hides the specific Flipt API. This allows for using environment variable based feature flags in local development and/or when flipt is not available.
+
+The recommended APIs for use in the backend are `GetBooleanFlagForUser` and `GetVariantFlagForUser`. The feature flag service object sets the `entityID` for those requests to the ID of the user making the request. It populates some default information in the context, including:
+  * The application name (mil, office, admin)
+  * If the user is an admin user, service member, or office user
+  * The [permissions](../../backend/guides/roles-and-permissions.md) of the user
+
+As we get more experience, we can add more information to the default context of a user.
+
+In addition, the API allows the caller to provide additional context that could be used for a specific feature flag.
+
+For feature flags that are not specific to a particular user, use the `GetBooleanFlag` and `GetVariantFlag` APIs for full customization of the `EntityID` and `Context`.
+
+### MilMove Backend Feature Flag Swagger API
+
+The `GetBooleanFlagForUse` and `GetVariantFlagForUser` APIs are exposed in the internal API via the `/feature-flags/user-boolean/{key}` and `/feature-flags/user-variant/{key}` endpoints so that the frontend can query feature flag status.
```

Comment:
```suggestion
The `GetBooleanFlagForUse` and `GetVariantFlagForUser` APIs are exposed in the internal API via the `/feature-flags/user-boolean/{key}` and `/feature-flags/user-variant/{key}` endpoints so that the frontend can query feature flag status. These endpoints have been added in [transcom/mymove#11330](https://github.com/transcom/mymove/pull/11330).
```